test/functional/lua/iter_spec.lua

```diff
@@ -139,6 +139,8 @@ describe('vim.iter', function()
 
   it('rev()', function()
     eq({ 3, 2, 1 }, vim.iter({ 1, 2, 3 }):rev():totable())
+    eq({ 3, 2 }, vim.iter({ 1, 2, 3 }):rev():take(2):totable())
```

Comment:
Thank you for adding a test case! I think these cases belong in the `take()` test block rather than `rev()` though.

Reply:
Moved!